### DISCUSSION TITLE
Follow creators for new projects

### DISCUSSION THREAD

**Discussion by CodeF53** [modrinth/.github#73] [Feature requests]
**Is your feature request related to a problem? Please describe.**
I have seen a few people create a lot of mods that interest me, so would like to be notifed about new mods they add to the platform

**Describe the solution you'd like**
Add a follow button
![image](https://user-images.githubusercontent.com/37855219/155260760-e6623ead-9a30-4eb6-9fc1-74f7445d1640.png)

**Describe alternatives you've considered**
Users could join mod devs discords, but not all mod devs have a discord.

**Comment by somyk33** [comment]
I support the idea. I've often found myself wanting to review the mods of certain creators. But to do this, you first need to find one of their mods and then go to the author's page.
It would also be great if there was a search for authors, but first of all, I wish Modrinth developers would implement the opportunity to subscribe to authors.

**Reply by N0tHorizon** [reply]
Agree, same idea here
